client/layout/masterbar/logged-in.jsx

```diff
@@ -150,11 +158,6 @@ class MasterbarLoggedIn extends Component {
 	clickMySites = () => {
 		this.props.recordTracksEvent( 'calypso_masterbar_my_sites_clicked' );
 
-		if ( this.props.isMobileGlobalNavVisible ) {
```

Comment:
Can we now remove `isMobileGlobalNavVisible` props?

Reply:
Along with a few others https://github.com/Automattic/wp-calypso/pull/92472/commits/c9df61847bb4e1408051aab562afb124b605b6ad